src/lib/ledgers/five-bells-ledger.js

```diff
@@ -38,33 +39,48 @@ FiveBellsLedger.prototype.getState = function (transfer) {
 }
 
 FiveBellsLedger.prototype.putTransfer = function * (transfer) {
+  const updatedTransfer = yield this._request({
+    method: 'put',
+    uri: transfer.id,
+    body: transfer
+  })
+  updateTransfer(transfer, updatedTransfer)
+}
+
+FiveBellsLedger.prototype.putTransferFulfillment = function * (transfer, execution_condition_fulfillment) {
+  const updatedTransfer = yield this._request({
+    method: 'put',
```

Comment:
Discussed with Stefan and we think it is better to use PUT for this. I have a ticket to update the ledger.